squid/tests/common.py

```diff
@@ -0,0 +1,79 @@
+# (C) Datadog, Inc. 2010-2016
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+import os
+from datadog_checks.utils.common import get_docker_hostname
+
+CHECK_NAME = "squid"
+HERE = os.path.dirname(os.path.abspath(__file__))
+HOST = get_docker_hostname()
+PORT = '3128'
+URL = 'http://{0}:{1}/squid-internal-mgr/counters'.format(HOST, PORT)
+OK = 0
+CRIT = 1
+SERVICE_CHECK = "squid.can_connect"
+CHECK_CONFIG = {
```

Comment:
can we make this a `instance()` fixture instead?

Reply:
also you should specify `instance['host'] = HOST` otherwise `parse_instance()` will assume `localhost`, reason why tests are failing